### DISCUSSION TITLE
HARD DRIVE

### DISCUSSION THREAD

**Discussion by astig121** [Chia-Network/chia-blockchain#5152] [Support]
hello fellow chia miners, just need to ask , is a 14tb HDD purple surveillance , will work in chia miner ? anyone tried?  thanks

**Comment by keeperlink** [comment]
Practically any HDD will do the job. After plotting is done, harvesting will use HDD very lightly

**Comment by l69zeroday94** [comment]
Hello guys, any opinion on this,
HP Proliant DL380 G6 - 98GB RAM, 4TB HDD (8X 500 GB)
Also on farm 4x rigs with ryzen 5x4, 32GBx4, 30TB SSD
How much can we expect from this monthly? I will appreciate any suggestions
for farming.

Best Regards



ned, 16. svi 2021. u 02:31 keeperlink ***@***.***> napisao je:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
